lib/frp/src/core/node/class.rs

```diff
@@ -0,0 +1,293 @@
+//! Root module for FRP node related abstractions.
+
+use crate::prelude::*;
+use crate::debug::*;
+use crate::node::id::*;
+use crate::node::label::*;
+use crate::data::*;
+
+
+
+// ===============
+// === DynNode ===
+// ===============
+
+/// Type level association between FRP data and dynamic node type. The associated `DynNode` type can
+/// differ depending on whether it is an event or behavior node, as they provide different APIs.
+/// For example, behaviors allow lookup for the current value, which does not make sense in case
+/// of events.
+pub trait KnownDynNode {
+    /// The node storage type.
+    type DynNode: DynNodeBounds + CloneRef;
+}
+
+/// Accessor.
+pub type DynNode<T> = <T as KnownDynNode>::DynNode;
```

Comment:
The description is provided on `KnownDynNode`. This is only accessor. I will add info to look there.

Reply:
```
/// Type level association between FRP data and dynamic node type. The associated `DynNode` type can
/// differ depending on whether it is an event or behavior node, as they provide different APIs.
/// For example, behaviors allow lookup for the current value, which does not make sense in case
/// of 
```
Event with that doc I still don't know what is "dynamic node type", I only know that it may differ for event and behavior nodes.

Reply:
I fixed that. I made it much better. Is it OK now?